.circleci/config.yml

```diff
@@ -13,10 +13,10 @@
 #
 #   * TERMINUS_MACHINE_TOKEN
 #
-# It also depends on the following environment variables declared in docker/drupal.env:
+# It also depends on the following environment variables declared in .env/drupal.env:
 #
 #   * TERMINUS_SITE
-#   * TERMINUS_SOURCE_ENVIRONMENT
+#   * TERMINUS_ENV
```

Comment:
Is there a compelling reason to rename this variable?  If not, I'd prefer to leave it as `TERMINUS_SOURCE_ENVIRONMENT` for backward compatibility reasons.